docs/machine-learning/tutorials/phone-calls-anomaly-detection.md

```diff
@@ -0,0 +1,261 @@
+---
+title: 'Tutorial: Detect anomalies in phone calls'
+description: Learn how to build an anomaly detection application for time series data. This tutorial creates a .NET Core console application using C# in Visual Studio 2019.
+ms.date: 12/04/2020
+ms.topic: tutorial
+ms.custom: mvc, title-hack-0612
+#Customer intent: As a developer, I want to use ML.NET in a time-series anomaly detection for number of phone calls so that I can analyze the data for anomaly points to take the appropriate action.
+---
+# Tutorial: Detect anomalies in time series with ML.NET
+
+Learn how to build an anomaly detection application for time series data. This tutorial creates a .NET Core console application using C# in Visual Studio 2019.
+
+In this tutorial, you learn how to:
+> [!div class="checklist"]
+>
+> * Load the data
+> * Detect period for a time series
+> * Detect anomaly for a periodical time series
+
+You can find the source code for this tutorial at the [dotnet/samples](https://github.com/dotnet/samples/tree/master/machine-learning/tutorials/ProductSalesAnomalyDetection) repository.
```

Comment:
Wrong link?

Reply:
Once the PR in the samples repo is merged, then this link will work.